snapshots/snapshots-annotations/src/main/kotlin/com/emergetools/snapshots/annotations/EmergeSnapshotConfig.kt

```diff
@@ -0,0 +1,17 @@
+package com.emergetools.snapshots.annotations
+
+/**
+ * Emerge-specific snapshot configuration options for the annotated Preview.
+ */
+@Target(AnnotationTarget.FUNCTION)
+annotation class EmergeSnapshotConfig(
+  /**
+   * The precision level for the snapshot comparison, with lower values allowing for more
+   * room for visual difference.
```

Comment:
Nit: perhaps an explanation the values go from 0.0f to 1.0f

Reply:
Will do, can restrict with an annotation